### PR TITLE
Add Onboarding tips component

### PR DESCRIPTION
**Pull request by pads**
Ignore the font awesome icon - I am not using the pro version in the example.

onboading_tip

### CODE REVIEW

File: assets/stylesheets/_bootstrap.scss [hunk 1]
```diff
@@ -35,6 +35,7 @@
 @import "bootstrap/alerts";
 @import "bootstrap/wells";
 @import "bootstrap/panels";
+@import "bootstrap/tips";
```
Comment (danielmatthew) [inline assets/stylesheets/_bootstrap.scss]: Nice one 👍

File: assets/stylesheets/bootstrap/_tips.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  color: $yellow;
+}
+
+.c-tip__title {
+  display: block;
+  font-weight: 600;
+}
+
+.c-tip__content {
+  margin-bottom: 0 !important;
```
Comment (danielmatthew) [inline assets/stylesheets/bootstrap/_tips.scss]: Is `!important` really needed?

**Comment by danielmatthew** [comment]
I wonder if the icon could go up to 3em? (Halve the margin again ;) )

**Comment by jeremybaines** [comment]
@pads @danielmatthew Icon should have been 'light' do we need to update boot strap TDL?

**Pull request merged**